loopy/kernel/function_interface.py

```diff
@@ -449,22 +419,32 @@ def with_target_if_not_None(dtype):
 
         return self.copy(arg_id_to_dtype=new_arg_id_to_dtype)
 
-    def with_hw_axes_sizes(self, global_size, local_size):
```

Comment:
There were no users of this method, got rid of it. (As there wasn't a need of specializing a callable based on the caller's execution grid).